### ISSUE TITLE
Respawn point

### ISSUE THREAD

**Issue by Vanikoro** [CryptechTest/cryptech#10]
add a way to change the spawn point

**Comment by SeqSEE** [comment]
Currently there are 2 ways to set a players spawn point.

/sethome 

and from the Unified  Inventory menu
![image](https://user-images.githubusercontent.com/15011102/227429425-a47b986c-242f-4301-9175-f8e039475ccf.png)
![image](https://user-images.githubusercontent.com/15011102/227429477-4adfddc9-b855-4b3d-9fc4-4aaeabc1ee25.png)

I think we should remove the command, and instead leave just the way that is in Unified Inventory for now. 

I believe there is a bug however in `ctg_world` mod, which incorrectly spawns the player at spawn even if they have set a home.

**Comment by Vanikoro** [comment]
yes you can set up a couple of homes, but I mean the respawn point when you die, not respawning in space, over and over again, unless that's the idea

**Comment by SeqSEE** [comment]
> yes you can set up a couple of homes, but I mean the respawn point when you die, not respawning in space, over and over again, unless that's the idea

Yeah, that was the part I said at the end.
